happypose_ros/test/happypose_testing_utils.py

```diff
@@ -391,6 +417,40 @@ def publish_image(
         self._cam_pubs[cam][0].publish(img_msg)
         self._cam_pubs[cam][1].publish(info_msg)
 
+        # TODO: refactor this
+        if depth is not None:
+            height = depth.shape[0] if height == 0 else height
+            width = depth.shape[1] if width == 0 else width
```

Comment:
We could make the assumption that depth image has to be the same size as rgb so maybe this should be replaced with `assert bgr.shape == depth.shape`